<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OffsetsControl</class>
 <widget class="QWidget" name="OffsetsControl">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1642</width>
    <height>857</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,0">
   <item>
    <widget class="QGroupBox" name="groupBox_CPUVoltageOffsets">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="title">
      <string>CPU Voltage</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <property name="topMargin">
         <number>30</number>
        </property>
        <property name="bottomMargin">
         <number>11</number>
        </property>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_cpuVoltageCore">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_cpuVoltageCore">
              <item>
               <widget class="QLabel" name="label_cpuVoltageCore">
                <property name="text">
                 <string>Core [mW]</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSlider" name="horizontalSlider_cpuVoltageCore">
                <property name="minimum">
                 <number>-500</number>
                </property>
                <property name="maximum">
                 <number>500</number>
                </property>
                <property name="singleStep">
                 <number>10</number>
                </property>
                <property name="pageStep">
                 <number>20</number>
                </property>
                <property name="orientation">
                 <enum>Qt::Orientation::Horizontal</enum>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QLCDNumber" name="lcdNumber_cpuVoltageCore">
              <property name="segmentStyle">
               <enum>QLCDNumber::SegmentStyle::Flat</enum>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_cpuVoltageUncore">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_cpuVoltageUncore">
              <item>
               <widget class="QLabel" name="label_cpuVoltageUncore">
                <property name="text">
                 <string>Uncore [mW]</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSlider" name="horizontalSlider_cpuVoltageUncore">
                <property name="minimum">
                 <number>-500</number>
                </property>
                <property name="maximum">
                 <number>500</number>
                </property>
                <property name="singleStep">
                 <number>10</number>
                </property>
                <property name="pageStep">
                 <number>20</number>
                </property>
                <property name="orientation">
                 <enum>Qt::Orientation::Horizontal</enum>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QLCDNumber" name="lcdNumber_cpuVoltageUncore">
              <property name="segmentStyle">
               <enum>QLCDNumber::SegmentStyle::Flat</enum>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_cpuVoltageCache">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_cpuVoltageCache">
              <item>
               <widget class="QLabel" name="label_cpuVoltageCache">
                <property name="text">
                 <string>Cache [mW]</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSlider" name="horizontalSlider_cpuVoltageCache">
                <property name="minimum">
                 <number>-500</number>
                </property>
                <property name="maximum">
                 <number>500</number>
                </property>
                <property name="singleStep">
                 <number>10</number>
                </property>
                <property name="pageStep">
                 <number>20</number>
                </property>
                <property name="orientation">
                 <enum>Qt::Orientation::Horizontal</enum>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QLCDNumber" name="lcdNumber_cpuVoltageCache">
              <property name="segmentStyle">
               <enum>QLCDNumber::SegmentStyle::Flat</enum>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_cpuVoltageGPU">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_cpuVoltageGPU">
              <item>
               <widget class="QLabel" name="label_cpuVoltageGPU">
                <property name="text">
                 <string>GPU [mW]</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSlider" name="horizontalSlider_cpuVoltageGPU">
                <property name="minimum">
                 <number>-500</number>
                </property>
                <property name="maximum">
                 <number>500</number>
                </property>
                <property name="singleStep">
                 <number>10</number>
                </property>
                <property name="pageStep">
                 <number>20</number>
                </property>
                <property name="orientation">
                 <enum>Qt::Orientation::Horizontal</enum>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QLCDNumber" name="lcdNumber_cpuVoltageGPU">
              <property name="segmentStyle">
               <enum>QLCDNumber::SegmentStyle::Flat</enum>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_cpuVoltageAnalogIO">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_cpuVoltageAnalogIO">
              <item>
               <widget class="QLabel" name="label_cpuVoltageAnalogIO">
                <property name="text">
                 <string>AnalogIo [mW]</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSlider" name="horizontalSlider_cpuVoltageAnalogIO">
                <property name="minimum">
                 <number>-500</number>
                </property>
                <property name="maximum">
                 <number>500</number>
                </property>
                <property name="singleStep">
                 <number>10</number>
                </property>
                <property name="pageStep">
                 <number>20</number>
                </property>
                <property name="orientation">
                 <enum>Qt::Orientation::Horizontal</enum>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QLCDNumber" name="lcdNumber_cpuVoltageAnalogIO">
              <property name="segmentStyle">
               <enum>QLCDNumber::SegmentStyle::Flat</enum>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_cpuVoltagePreset">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_cpuVoltagePreset">
            <item>
             <widget class="QLabel" name="label_cpuVoltagePreset">
              <property name="text">
               <string>Preset undervolting  configuration</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="comboBox_cpuVoltagePreset"/>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="horizontalSpacer_cpuVoltagePreset">
            <property name="orientation">
             <enum>Qt::Orientation::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_cpuVoltageCancel">
            <property name="text">
             <string>Cancel</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_cpuVoltageApply">
            <property name="text">
             <string>Apply</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_GPUOffsets">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="title">
      <string>GPU Clock</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_8">
      <property name="topMargin">
       <number>30</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_GPUClockOffset">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_GPUClockOffset">
          <item>
           <widget class="QLabel" name="label_GPUClockOffset">
            <property name="text">
             <string>GPU Clock Offset [MHz]</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSlider" name="horizontalSlider_GPUClockOffset">
            <property name="minimum">
             <number>-500</number>
            </property>
            <property name="maximum">
             <number>500</number>
            </property>
            <property name="singleStep">
             <number>10</number>
            </property>
            <property name="pageStep">
             <number>20</number>
            </property>
            <property name="orientation">
             <enum>Qt::Orientation::Horizontal</enum>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLCDNumber" name="lcdNumber_GPUClockOffset">
          <property name="segmentStyle">
           <enum>QLCDNumber::SegmentStyle::Flat</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_GPUMemoryOffset">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <item>
           <widget class="QLabel" name="label_GPUMemoryOffset">
            <property name="text">
             <string>Memory Clock Offset</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSlider" name="horizontalSlider_GPUMemoryOffset">
            <property name="minimum">
             <number>-500</number>
            </property>
            <property name="maximum">
             <number>500</number>
            </property>
            <property name="singleStep">
             <number>10</number>
            </property>
            <property name="pageStep">
             <number>20</number>
            </property>
            <property name="orientation">
             <enum>Qt::Orientation::Horizontal</enum>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLCDNumber" name="lcdNumber_GPUMemoryOffset">
          <property name="segmentStyle">
           <enum>QLCDNumber::SegmentStyle::Flat</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_GPUPreset">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_GPUPreset">
          <item>
           <widget class="QLabel" name="label_GPUPreset">
            <property name="text">
             <string>Preset offsets  configuration</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="comboBox_GPUPreset"/>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="horizontalSpacer_GPUPreset">
          <property name="orientation">
           <enum>Qt::Orientation::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_GPUPreset_ApplyCancel">
          <item>
           <widget class="QPushButton" name="pushButton_GPUCancel">
            <property name="text">
             <string>Cancel</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_GPUApply">
            <property name="text">
             <string>Apply</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer_OffsetsControl">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
